tests/AbstractGenerator.php

```diff
@@ -34,7 +34,7 @@ public function __construct(array $targetNs, array $aliases = array(), $tmp = nu
         $this->phpDir = "$tmp/php";
         $this->jmsDir = "$tmp/jms";
 
-        $this->namingStrategy = new ShortNamingStrategy();
+        $this->namingStrategy = new VeryShortNamingStrategy();
```

Comment:
why changing this?

Reply:
The test case using the OTA schema was resulting in so many nested directories that mkdir was failing for being "too long". I put the strategy in the test directory as a workaround.

Reply:
```
GoetasWebservices\Xsd\XsdToPhp\PathGenerator\PathGeneratorException: Can't create the 'C:\Users\.........\AppData\Local\Temp/php/ota/RFPResponseType/RFPResponseSegmentsAType/RFPResponseSegmentAType/SitesAType/SiteAType/DatesAType/DateAType/RoomBlockAType/StayDatesAType/StayDateAType/StayDateRoomsAType/StayDateRoomAType/RatesAType' directory: 'mkdir(): Filename too long' in C:\server\root\oss\xsd2php\src\Php\PathGenerator\Psr4PathGenerator.php:19
```